### PR TITLE
🌱 v2.25.0

### PR DESCRIPTION
**Pull request by luxonauta**
-   Removed the Soft variant;
-   Introduced new colors settings for Editor Marker Navigation, Peek View Colors, Command Center, and Notification Colors, enhancing customization and visibility.
-   Significant adjustments to various editor components, including the Diff Editor, Inline Chat, Editor Widgets, Editor Suggest Widget, Editor Hover Widget, and Debug Exception Widget, ensuring a more cohesive and consistent experience;
-   Improvements to various meta scopes, increasing the accuracy and clarity of the interface;
-   Addressed various edge cases for different languages and log file entries, ensuring expanded compatibility and functionality.

**Note:** This update brings the Dracula Theme closer to what we have in Dracula PRO and thus facilitates support for both; it also prepares the ground for us to introduce the Light variant for the Classic in the future.

### CODE REVIEW

**Comment by dsifford** [comment]
@luxonauta I do not agree with your aggressiveness here. Who approved this?

**Comment by luxonauta** [comment]
Hey 👋🏻 @dsifford, Zeno and I have been working on these updates for PRO for a while, and we wanted to port them to Classic as quickly as possible to bring the Dracula Classic theme closer to what we have in Dracula PRO.

**Comment by dsifford** [comment]
I do not agree here. I emailed zeno just now and I will say that if this is the practice that you are going to use moving forward then I'm forking this project and leaving. This is way too aggressive and is not in the spirit of OSS.

File: bun.lockb
Comment (dsifford) [inline bun.lockb:1]: @luxonauta Why would you even do this? I don't have bun and don't plan to download bun. Most contributors likely don't have bun. Please revert this.

File: .editorconfig
Comment (dsifford) [inline .editorconfig:1]: Please revert this.

File: .prettierrc.toml [hunk 1]
```diff
@@ -6,9 +6,4 @@ trailingComma = 'all'
 [[overrides]]
   files = '*.js'
   [overrides.options]
-    trailingComma = 'es5'
-
-[[overrides]]
-  files = '*.{yml,yaml}'
-  [overrides.options]
-    tabWidth = 2
```
Comment (dsifford) [inline .prettierrc.toml:14]: This change has nothing to do with the task here and should have been done in a separate PR.

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
-    "[yaml]": {
+    "files.eol": "\n",
+    "[md]": {
+        "editor.wordWrap": "bounded",
+        "editor.wordWrapColumn": 75
+    },
+    "[yml]": {
+        "editor.insertSpaces": true,
         "editor.renderWhitespace": "boundary",
+        "editor.tabSize": 2
```
Comment (dsifford) [inline .vscode/settings.json:11]: This change has nothing to do with the task here and should have been done in a separate PR.

File: src/dracula.yml
Comment (dsifford) [inline src/dracula.yml:1]: Because you changed the formatting rules, this changeset has become effectively non-reviewable. How do you expect people to see what you've actually done here?

**Comment by luxonauta** [comment]
Hey 👋🏻 @dsifford,

I understand your concern and would like to clarify a few points. 🧩

From our perspective, the repo seemed abandoned, with several issues dating back two years. Despite trying to email you and commenting on several PRs and Issues, we are still waiting to receive a response (_which is not your obligation,_ and we understand and respect your time).

However, the last commit was on _July 15, 2023_. One year later, with several updates to the PRO, **we decided in December last year** to bring Classic closer to what we have in Dracula PRO to facilitate support for both and **actively support them closely.** 😅

I agree with you, and I'll exercise more caution and thoughtfulness when making changes like this one. If you agree, I will also look for your review on changes like this one.

Our shared goal is to collaborate and improve the project constructively.

✅ Also, I'll make sure to open separate PRs with the changes you suggested here as comments.

**Comment by dsifford** [comment]
A few things here.

1. You have never emailed me.
2. A theme is not something that changes often. Issue count should not be treated as abandonment. Lots of those issues were single-user reports requesting jarring changes. I was waiting until those issue received traction before moving forward on them.
3. Dracula Pro is a fork of this repository. I'm flattered. But I do not agree with just taking the changes you made to that fork and blasting it back here without code review. This is not how software works. This feels to me like a hostile takeover and I very much do not appreciate almost a decade of my work getting steamrolled in one shot.

I understand that you're enthusiastic. That's very nice to see. But this is not how OSS works. Your changes here I do not agree with and unless they are rolled back and made in a way that is _reviewable_. Then I'm forking this project and leaving.

@zenorocha I would love to hear your thoughts here. Do you agree with how this change was performed?

File: bun.lockb
Comment (dsifford) [reply bun.lockb:1]: @zenorocha

**Comment by luxonauta** [comment]
I'm sorry if there was any miscommunication. We attempted multiple times to reach out and get feedback on the PRs and issues, but I understand there may have been some misunderstandings. 😥

Our intention has always been to collaborate and improve the project constructively. I appreciate your work and dedication over the years, and I am committed to respecting and working together for the benefit of the project.

> Also, the changes were made to contribute positively, and there was no intention to pressure or disrespect you.

✅ I am available to discuss and review any necessary changes to ensure we are all aligned.

**Comment by dsifford** [comment]
> We attempted multiple times to reach out and get feedback on the PRs and issues,

Please show me.

**Comment by luxonauta** [comment]
I believe it will be more beneficial _and polite_ to iterate together from now on than to continue this discussion.

✅ _I am available to discuss and review any necessary changes to ensure we are all aligned._ 😅

**Comment by GoodbyeNJN** [comment]
> believe it will be more beneficial and polite to iterate together from now on than to continue this discussion.

Since there has been quite a bit of negative feedback and the change does not follow the contribution process, the immediate priority should be to roll back the code and release a new version to avoid disrupting more users.

As for iteration and discussion, that is something that should be considered after this.

**Comment by luxonauta** [comment]
I reverted the recent changes and will open separate PRs for review. Thank you all for the discussion and feedback! 🧩

**Comment by austinried** [comment]
Why was this allowed to happen? Kind of shocking that someone can just merge in such a drastic change like this with no review and break millions of installs, especially with the [recent scare related to typo-squatting this theme specifically](https://www.bleepingcomputer.com/news/security/malicious-vscode-extensions-with-millions-of-installs-discovered/).

The broken version is also still up as the most recent one on [OpenVSX](https://open-vsx.org/extension/dracula-theme/theme-dracula).

**Comment by wenfangdu** [comment]
@luxonauta Can you please reopen my issues that were closed by this PR?
- https://github.com/dracula/visual-studio-code/issues/183
- https://github.com/dracula/visual-studio-code/issues/184
- https://github.com/dracula/visual-studio-code/issues/234

And also other people's issues that were closed by this PR.
